src/DetailsPage/index.js

```diff
@@ -110,66 +110,73 @@ class DetailsPage extends Component {
       });
     }
 
-    this.setState({ showLoader: true, fetchComplete: false, loaderHint: 'Updating Tangle' });
-    const response = await appendItemChannel(metadata, this.props, this.documentExists, status);
+    setShowLoader(true);
+    setFetchComplete(false);
+    setLoaderHint('Updating Tangle');
+    const infos = { item, items, project, match }
+    const response = await appendItemChannel(metadata, infos, documentExists, status);
```

Comment:
While testing I noticed that the `metadata` value is empty, despite the fact it was set on line 193. 
This might require a more intensive debugging, but a quick and dirty solution would be to send **metadata** value as a second parameter to the `appendToItem` function, so it will look as follows: 
```
const appendToItem = async (status, meta) => {
  ...
  const response = await appendItemChannel(meta, infos, documentExists, status);
  ...
}

const onUploadComplete = async meta => {
   ...
  appendToItem(null, meta);
  ...
};
```